rosdep/python.yaml

```
@@ -6719,6 +6719,11 @@ python3-pyrr-pip:
   ubuntu:
     pip:
       packages: [pyrr]
+python3-pyside2.qtopengl:
+  ubuntu:
```

Fedora 32 https://fedora.pkgs.org/32/fedora-x86_64/python3-pyside2-5.13.2-2.fc32.x86_64.rpm.html
Fedora 33 https://fedora.pkgs.org/33/fedora-x86_64/python3-pyside2-5.15.0-4.fc33.x86_64.rpm.html
I think this might do:
```suggestion
  debian: [python3-pyside2.qtopengl]
  fedora: [python3-pyside2]
  gentoo: [dev-python/pyside2]
  ubuntu:
```

---

Thanks for your help, I was a little bit blocked since I only know the Ubuntu packages.